extension/scripts/global/functions/torn.js

```diff
@@ -260,3 +260,7 @@ function getPage() {
 function isCaptcha() {
 	return !!document.find(".captcha");
 }
+
+function darkMode() {

```

Comment:
Should have a proper name, like `hasDarkMode`.

Reply:
Changed.